tests/complex/virtwhotest.py

```diff
@@ -106,7 +141,8 @@ def run_virtwho(self, args, grab_stdout=False, background=False):
         code = None
         data = None
         socket.setdefaulttimeout(1)
-        self.process = Process(target=virtwho.main.main)
+        from virtwho.__main__ import main as main_main
+        self.process = Process(target=main_main)
```

Comment:
Non-nacking comment: This tests slightly less of our codebase as it is a different entry point than that used by the installed virt-who script.